notifications_utils/template.py

```diff
@@ -372,17 +371,15 @@ def __init__(
         self,
         template,
         values=None,
-        polygons=None,
-        areas=None,
+        areas=[],
+        polygons=[],
```

Comment:
You should [avoid using mutable types as default arguments](https://florimond.dev/blog/articles/2018/08/python-mutable-defaults-are-the-source-of-all-evil/) unless you really dislike your future self. Instead do:
```python
def __init__(areas=None):
    self.areas = areas or []
```

Reply:
a66559ec6728c49ee78fe3e0fc849afa517c207d